frontend/coprs_frontend/coprs/templates/coprs/_coprs_forms.html

```diff
@@ -159,7 +159,7 @@ <h3 class="panel-title">{{ counter('instructions') }}. Other options</h3>
     {{ render_field(form.repo_priority,
         class="short-input-field",
         placeholder='Optional',
-        info='Set the priority value of this repository. Defaults to 99.') }}
+        info='Use the priority=&lt;INT&gt; config option for repositories in this project, see man dnf.conf(5) for more info. Defaults to 99.') }}
```

Comment:
Last small wish (I changed my opinion) - I'd rather remove "Defaults to 99." part (this info is both in dnf.conf(5) and our default is actually None; plus I don't think we should unnecessarily raise eyebrows).

Reply:
np